### DISCUSSION TITLE
Conditionally exclude multiple properties

### DISCUSSION THREAD

**Discussion by ju5t** [spatie/laravel-data#522] [Ideas]
There is a feature we could use, but before creating a PR, I thought I'd bring it up first. Every now and then we need to conditionally exclude multiple properties at once. 

We could use something like `excludeManyWhen(['a', 'b', 'c',], fn () => true !== false)`.

Would you consider something like this?

**Comment by rubenvanassche** [comment]
Why not update the method signature to `public function includeWhen(string|array $include, bool|Closure $condition):`. That way we don't need to introduce more methods. I would accept this PR btw.
